### DISCUSSION TITLE
Gradual SV changes for taiko

### DISCUSSION THREAD

**Discussion by HexaMaster** [ppy/osu#17844] [Ideas] [answered]
Adding gradual slider velocity (note speed) changes is very common in taiko maps, and this is currently done by adding a new inherited point on every note in the gradual change and setting the slider velocity multiplier to a slightly higher/lower value each time. However, especially with the addition of longer decimal values for slider velocity, it would be very helpful to have a way to make this process less tedious. For example, the ability to set a start and end point for a slider velocity change with all notes in the area gradually going from the starting value to the ending one. This could also work with gradual audio changes. Even though the current way of doing this works, gradually changing note speed in an area with many hundreds of hitcircles can take an hour or more and this feature could simplify this process to less than a minute.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/9784
